### DISCUSSION TITLE
Data labels not having enough space

### DISCUSSION THREAD

**Discussion by marks-kr** [patrykandpatrick/vico#819] [Questions]
### Question

The data labels in my bar chart look normal when their size is 8sp, but I just had an ask to increase them to 10sp and that causes them to overlap the bars: 
![Capture](https://github.com/user-attachments/assets/dd50a63d-6d8b-4845-ae4c-7be7e3c7a82d)
I tried adding a value for the labels' padding attribute but that caused the same issue even when it's set to 0dp vertical and the text size to 8sp.  What do I need to do to ensure the labels are given enough room between the top of the tallest bars and the top of the chart area?

### Vico version(s)

2.0.0-alpha.18

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
Hello @marks-kr, please see #448 and #573.
